http-client/src/test/java/io/airlift/http/client/TestHttpClientBinder.java

```diff
@@ -125,20 +118,16 @@ public void testBindingMultipleFiltersAndClients()
             throws Exception
     {
         Injector injector = new Bootstrap(
-                new Module()
-                {
-                    @Override
-                    public void configure(Binder binder)
-                    {
-                        httpClientBinder(binder).bindHttpClient("foo", FooClient.class)
-                                .withFilter(TestingRequestFilter.class)
-                                .withFilter(AnotherHttpRequestFilter.class)
-                                .withTracing();
-
-                        HttpClientBindingBuilder builder = httpClientBinder(binder).bindHttpClient("bar", BarClient.class);
-                        builder.withFilter(TestingRequestFilter.class);
-                        builder.addFilterBinding().to(AnotherHttpRequestFilter.class);
-                    }
+                binder -> {
+                    httpClientBinder(binder)
+                            .bindHttpClient("foo", FooClient.class)
+                            .withFilter(TestingRequestFilter.class)
+                            .withFilter(AnotherHttpRequestFilter.class)
+                            .withTracing();
+
+                    HttpClientBindingBuilder builder = httpClientBinder(binder).bindHttpClient("bar", BarClient.class);
```

Comment:
While we're changing this, let's get rid of the builder. Users of the binder shouldn't normally see the type `HttpClientBindingBuilder`